src/providers/postgres/qgspostgresprovider.cpp

```diff
@@ -1123,8 +1123,8 @@ bool QgsPostgresProvider::loadFields()
       else if ( fieldTypeName == QLatin1String( "name" ) )
       {
         fieldSubType = QVariant::String;
-        // "name" type lenght is 64 according to: SELECT typlen FROM pg_type WHERE typname = 'name';
-        fieldSize = 64;
+        // "name" type lenght is 63
```

Comment:
typo - switch to length

Reply:
```suggestion
        // "name" type length is 63
```

Reply:
@stev-0 FYI you can use GitHub "Suggestions" for this kind of comment ^.

And thanks for raising this :+1: